assets/src/js/Components/archivePicker/ArchivePickerList.js

```diff
@@ -47,18 +48,32 @@ export default function ArchivePickerList() {
                 </div>
               )}
               <img
-                className={classNames({'picker-selected': selectedImages[image.id] && true})}
+                className={classNames({'picker-selected': selectedImages[image.id]})}
                 srcSet={toSrcSet(sizes, {maxWidth: 900})}
                 title={title}
                 alt={alt}
                 width={200 * (original.width / original.height)}
                 height={200}
-                onClick={event => {
-                  // metaKey for Mac users
-                  dispatch({type: 'TOGGLE_IMAGE', payload: {image, multiSelection: (event.ctrlKey || event.metaKey)}});
+                {...(!bulkSelect) && {
+                  onClick: () => {
+                    // metaKey for Mac users
+                    dispatch({type: 'TOGGLE_IMAGE', payload: {image, multiSelection: (event.ctrlKey || event.metaKey)}});
+                  },
                 }}
                 role="presentation"
               />
+              {
+                bulkSelect && !wordpress_id && (
+                  <div
+                    role="button"
+                    aria-hidden="true"
+                    className={classNames('bulk-select-checkbox', {'is-checked': selectedImages[image.id]})}
+                    onClick={() => {
```

Comment:
I guess it's gonna be for the design team to decide, but maybe this should be implemented when clicking on the image rather than specifically on the checkbox?

Reply:
It makes more sense!